code/src/components/Artist.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+export const Artist = (props) => {
+
+	if (props.name.length === 1) {
+		return (
+			<div>
+				<a href={props.url[0]} alt=''>
+					<h3 className="artistsName artistName">{props.name[0]}</h3>
+				</a>
```

Comment:
`props.url[0]` and `props.name[0]` seems to be used more than once. Have you thought about creating constants for both of these?